packages/core/src/mutants/MutatorFacade.ts

```diff
@@ -15,7 +15,10 @@ export class MutatorFacade implements Mutator {
 
   public mutate(inputFiles: ReadonlyArray<File>): ReadonlyArray<Mutant> {
     const allMutants = this.pluginCreator.create(this.getMutatorName(this.options.mutator))
-      .mutate(inputFiles);
+      .mutate(
+        inputFiles,
+        typeof this.options.mutator !== 'string' ? this.options.mutator.babelPlugins : []
```

Comment:
This should not be done in the `Facade`, but in the `JavaScriptMutator` itself.

Reply:
I had to pass options somehow. Honestly no idea how can i do it other way